besu/src/main/java/org/hyperledger/besu/controller/BesuController.java

```diff
@@ -258,5 +271,19 @@ private Long readQbftStartBlockConfig(final QbftConfigOptions qbftConfigOptions)
 
       return startBlock;
     }
+
+    private boolean isCheckpointPoSBlock(final GenesisConfigOptions configOptions) {
+      UInt256 terminalTotalDifficulty = configOptions.getTerminalTotalDifficulty().get();
+      // this is currently a limitation that besu has in its codebase
```

Comment:
nit: What limitation? Comment isn't really describing why there's a problem. The exception message is clear, so maybe this comment is unnecessary. Otherwise, would be good to say what needs to happen to remove the limitation.